src/commands/Angles.cpp

```diff
@@ -80,6 +80,11 @@ Averager<double> AngleDistribution::setup(int argc, const char* argv[]) {
 void AngleDistribution::finish(const Histogram<double>& histogram) {
     auto max = *std::max_element(histogram.begin(), histogram.end());
 
+    if (max == 0) {
+        throw CFilesError("No angle corresponding to the '" + selection_.string() + "' selection found.");
+        max = 1;
```

Comment:
No need to set max to 1 after a throw, as the exception will return to the caller.